docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,167 @@
+
+<p align="center"><img src="../../getting-started/apm-overview.png" alt="APM dashboard"></p>
+
+## Requirements to enable the APM dashboard
+
+You have to enable metric graph and service graph generation on the Grafana backend so metrics are generated as traces are ingested.
+
+To use the APM dashboard, you need:
+
+* Tempo or Grafana Cloud Traces with the metrics generator enabled and configured to send data to a Prometheus compatible metrics
+* [Services graphs]({{< relref "../metrics-generator/service_graphs/#how-to-run" >}}), which are enabled by default in Grafana
+* [APM table enabled](https://grafana.com/docs/grafana/next/datasources/tempo/) within the service graph tab (use the `tempoApmTable` feature flag in Grafana to enable)
+* [Span metrics]({{< relref "../metrics-generator/span_metrics/#how-to-run" >}}) enabled in your Tempo data source configuration
+
+The APM dashboard can be generated with the metrics-generator or Grafana Agent.
+
+For information on how to configure these features, refer to the [Grafana Tempo data sources documentation](https://grafana.com/docs/grafana/next/datasources/tempo/).
+
+## APM dashboard
+
+Using this dashboard, you can see the top five spans by request rate. You can refine any of this data using the filters. Selecting any of the data points lets you see more specific data.
+
+The APM dashboard provides a span metrics visualization (APM table, screen section 2) and service graph (screen section 3). In addition, you can use the filters (screen section 1) to customize the data displayed.
+
+<p align="center"><img src="../apm-overview-numbered.png" alt="APM dashboard with numbered sections"></p>
+
+Any information in the APM table that has an underline can be selected to show more detailed information. You can also select any node in the service graph to display additional information. In the dashboard shown below, the `Ingester.QueryStream` span has a request rate of `144220.22` requests per second. The `cortex/IngesterQuery` span has the highest request rate.
+
+### Error rate example
+
+Let’s say we want to learn more about why `cortex.Ingester` has the highest error rates.
+Selecting the second row of the Error rate column displays details about the span metrics in a new window on the right side.
+
+<p align="center"><img src="../apm-error-rate-example.png" alt="APM error rate example"></p>
+
+The metrics query used to generate the data appears in the **Metrics browser** field.
+
+<p align="center"><img src="../apm-error-example-editor.png" alt="APM error example query editor"></p>
+
+## Span metrics table
+
+The span metrics, also called the APM table, are generated by the metrics-generator or the Grafana Agent. These metrics are created from ingested tracing data, including RED metrics.
+
+Span metrics generate two metrics:
+
+* A counter that computes requests
+* A histogram that tracks the distribution of durations of all requests per second
```

Comment:
```suggestion
* A histogram that tracks the distribution of durations of all requests
```

I think its not 'per second' is it? 
I think you just do distribution of request durations. The rate (requests per second) comes from looking at the change in the counter over time. 
But I may be totally wrong here :)